oshi-core/src/main/java/oshi/software/os/linux/LinuxProcess.java

```diff
@@ -103,25 +104,36 @@ public LinuxProcess(String name, String path, char state, int processID, int par
      * jiffies
      */
     private static void init() {
-        // Search through all processes to find the youngest one, with the
-        // latest start time since boot.
+        // To correlate a process start time in seconds with the same process
+        // start time in jiffies. We prefer the youngest (or close to it) which
+        // minimizes its up time (etime)
+        // Timeline:
+        // BOOT|<----jiffies---->|<----etime---->|NOW
+        // BOOT|<------------uptime------------->|NOW
 
-        // Iterate /proc/[pid]/stat checking the creation time (field 22,
-        // jiffies since boot) for the largest value
+        // To avoid having to check all processes we can just pick the highest
+        // PID. This will either be the youngest one or at least big enough.
+
+        // Get all the pid files (guaranteed to be digit-only filenames)
         File[] pids = ProcUtil.getPidFiles();
+        // Sort descending "numerically"
+        Arrays.sort(pids, (f1, f2) -> Integer.valueOf(f2.getName()).compareTo(Integer.valueOf(f1.getName())));

```

Comment:
Now it would be fair to note that this method is inefficient; it requires the String to integer coversion for every comparision, rather than converting the String array to integer, sorting, and then converting back only the needed integers.  I'll take the one-liner readability though.